django/utils/version.py

```diff
@@ -6,6 +6,9 @@
 
 from django.utils.regex_helper import _lazy_re_compile
 
+# Private, stable API for detecting the Python implementation.
+PYPY = sys.implementation.name == "pypy"
```

Comment:
small nit: this is not really "private", it is [PEP 421](https://peps.python.org/pep-0421/) and was added for python>=3.3. `sys.implementation` does not exist on python2

Reply:
"Private and stable" apply here to the `PYPY` variable, not to the `sys.implementation`.